bcos-gateway/bcos-gateway/libp2p/P2PMessage.h

```diff
@@ -175,6 +175,10 @@ class P2PMessage : public Message
     {
         return (m_ext & bcos::protocol::MessageExtFieldFlag::Response) != 0;
     }
+
+    // compress payload if payload need to be compressed
+    bool compress(std::shared_ptr<bytes> compressData);
```

Comment:
Rename to `tryCompress`.

Reply:
has rename this function name to tryCompress.